utils/http.py

```diff
@@ -12,3 +12,15 @@ def retrieve_json(url, timeout=DEFAULT_TIMEOUT, verify=True):
     r = requests.get(url, timeout=timeout, verify=verify)
     r.raise_for_status()
     return r.json()
+
+# Get expvar stats
+def get_expvar_stats(key, port):
+    try:
+        json = retrieve_json("http://127.0.0.1:{port}/debug/vars".format(port=port))
```

Comment:
We're not binding to `bind_host` on the dogstatsd6 side, but to localhost or `0.0.0.0`, so I think we should maybe keep it as `localhost`. But, I don't have a strong opinion either.